persistent/tests/test_persistence.py

```diff
@@ -1733,12 +1733,27 @@ def __repr__(self):
 
     def test_repr_oid_no_jar(self):
         p = self._makeOne()
-        p._p_oid = b'12345678'
+        p._p_oid = self._PACKED_OID
 
         result = self._normalized_repr(p)
         self.assertEqual(
             result,
-            "<persistent.Persistent object at 0xdeadbeef oid b'12345678'>")
+            "<persistent.Persistent object at 0xdeadbeef oid " + self._HEX_OID + ">")
+
+    def test_64bit_oid(self):
+        import struct
+        p = self._makeOne()
+        oid_value = 2 << 62
```

Comment:
1 << 63 would be clearer imho

Reply:
I was looking at this, wondering why 2**62 instead of 2**63, and then I realized << is not ** :D